bin/git-forgit

```diff
@@ -258,6 +267,10 @@ _forgit_get_single_file_from_add_line() {
     sed -e 's/^\"//' -e 's/\"$//'
 }
 
+_forgit_edit_add_file() {
+    $EDITOR "$(echo "$1" | _forgit_get_single_file_from_add_line)" >/dev/tty </dev/tty
```

Comment:
This has been implemented as a one-liner, whereas `_forgit_edit_diffed_file()` has split the nearly identical work into three (better readable) lines. We should transform this accordingly to have the same style in both functions.